### DISCUSSION TITLE
Plan for 2.2.0 release

### DISCUSSION THREAD

**Discussion by sandrotosi** [puddletag/puddletag#740] [General]
Hello folks,
quite a few changes have accumulated in the git repo since the last release (according to github, 167 commits!), so i think it's time to consider releasing a new version and given the number of changes, bump it to 2.2.0.

@corubba is there anything else you want to get in before the release?

thanks!

**Comment by sandrotosi** [comment]
apologies for now doing it before, tagging some of the main contributors/collaborators: @blueblots @pedrobaeza @bernd-wechner

**Reply by pedrobaeza** [reply]
Ok for me and don't worry. I was using git version.

**Reply by bernd-wechner** [reply]
Likewise, I just run off the github pull .... so I can fix issues on the fly ;-). It's getting better slowly, all the time thanks to everyone's efforts!

**Comment by corubba** [comment]
Nothing comes immediately to my mind. It already is a big release, no need to blow it up even more with last minute changes.

**Comment by sandrotosi** [comment]
Just cut 2.2.0: https://github.com/puddletag/puddletag/releases/tag/2.2.0 -- thanks everyone for your contributions!
